src/python_testing/matter_testing_infrastructure/data_model_xmls.gni

```diff
@@ -116,7 +121,6 @@ data_model_XMLS = [
   "${chip_root}/data_model/1.3/clusters/WindowCovering.xml",
   "${chip_root}/data_model/1.3/clusters/bridge-clusters-ActionsCluster.xml",
   "${chip_root}/data_model/1.3/clusters/bridge-clusters-BridgedDeviceBasicInformationCluster.xml",
-  "${chip_root}/data_model/1.3/clusters/cluster_ids.json",
```

Comment:
Was this json not used previously by anything that used `spec_parsing.py`, and that is why you have removed it without any other CI failing?

Reply:
yeah, it's just a list of clusters in json format - doesn't need to be part of the testing package.